script.js

```diff
@@ -1,5 +1,47 @@
+
 const playButton = document.querySelector('.play')
 const playMenu = document.querySelector('.gameButtons')
+const button1 = document.querySelector('.btn1')
+const button2 = document.querySelector('.btn2')
+const button3 = document.querySelector('.btn3')
+const hintButton = document.querySelector('.hintButton')
+const scoreBoxElem = document.querySelector('.scoreBox')
+const quoteElem = document.querySelector('.quote')
+const instructionsButton = document.querySelector('.instructionsButton')
+const instructions = document.querySelector('.modalContainer')
+const closeButton = document.querySelector('.closeButton')
+const timerDisplay = document.querySelector(".timerDisplay")
+const timerProgress = document.querySelector(".timerProgress")
+const resetBtn = document.querySelector('.reset')
+const gameOverModal = document.querySelector('.gameOverModal')
+const scoreSpan = document.querySelector('.score')
+let filmTitles = []
+let films = []
+let score = 0
+let countDownValue = 30
+let countDownInterval
+
+const fetchFilms = async () => {
+    const response = await fetch('./films.json')
+    const data = await response.json()
+    return data.films
+}
+
+const setupGame = async () => {
+    filmTitles = await fetchFilms()
+    films = await fetchFilms()
+    playButton.addEventListener('click', () => playGame(filmTitles, films))
+}
+
+const modal = () => {
+    if (instructions.style.display === 'none') {
+        instructions.style.display = 'flex';
+        instructions.style.zIndex = '1';
+    } else {
+        instructions.style.display = 'none';
+    }
+}
```

Comment:
This function is only called `modal`? Isn't that really confusing?

Do you know about F2 / Refactor to rename functions?